File: ls/server/src/main/java/org/opencds/cqf/cql/ls/server/event/DidChangeWatchedFilesEvent.java
package org.opencds.cqf.cql.ls.server.event;

import org.eclipse.lsp4j.DidChangeWatchedFilesParams;

public class DidChangeWatchedFilesEvent extends BaseEvent<DidChangeWatchedFilesParams> {
    public DidChangeWatchedFilesEvent(DidChangeWatchedFilesParams params) {
        super(params);
    }
}

File: ls/server/src/main/kotlin/org/opencds/cqf/cql/ls/server/event/DidChangeWatchedFilesEvent.kt
package org.opencds.cqf.cql.ls.server.event

import org.eclipse.lsp4j.DidChangeWatchedFilesParams

class DidChangeWatchedFilesEvent(params: DidChangeWatchedFilesParams) : BaseEvent<DidChangeWatchedFilesParams>(params)
